mittab/libs/data_import/import_teams.py

```diff
@@ -72,98 +91,100 @@ def import_teams(fileToImport):
-            deb1.save()
-        except:
-            team_errors.append(team_name + ': Unkown Error Saving Debater 1')
-            continue
-        if (not iron_man):
-            try:
-                deb2 = Debater(name = deb2_name, novice_status = deb2_status, phone = deb2_phone, provider = deb2_provider)
-                deb2.save()
-            except:
-                team_errors.append(team_name + ': Unkown Error Saving Debater 2')
-                team_errors.append('        WARNING: Debaters on this team may be added to database. ' +
-                                    'Please Check this Manually')
-                continue
-        
-        team = Team(name=team_name, school=team_school, seed=team_seed)
-        try:
+        if not duplicate:  # create new team
+            team = Team(name=team_name, school=team_school, seed=team_seed)
             team.save()
             team.debaters.add(deb1)
-            if (not iron_man):
+            if not iron_man:
                 team.debaters.add(deb2)
             else:
-                team_errors.append(team_name + ": Detected to be Iron Man - Still added successfully")
+                team_errors.append(team_name + ': Team is an iron-man, added successfully')
+            team.save()
+
+        else:  # update the team
+            team = Team.objects.get(name=team_name)
+            team.school = team_school
+            team.seed = team_seed
+
+            team.debaters.clear()
+            team.debaters.add(deb1)
+            if not iron_man:
+                team.debaters.add(deb2)
+            else:
+                team_errors.append(team_name + ': Team is an iron-man, added successfully')
             team.save()
-        except:
-            team_errors.append(team_name + ': Unknown Error Saving Team')
-            team_errors.append('        WARNING: Debaters on this team may be added to database. ' +
-                                'Please Check this Manually')
 
     return team_errors
 
-    
 
+def _create_status(status):
+    """Translates the string for varsity-novice status into MIT-TAB's integer pseudo-enum"""
+    if status == 'novice' or status == 'nov' or status == 'n':
+        return 1
+    else:
+        return 0
+
+
+def _create_seed(team_name, seed):
+    """Translates the string version of the seed into the pseudo-enum. Checks for duplicate free seeds and changes it
+    as necessary. Also notes that change so a message can be returned.
+    :type team_name: str
+    :type seed: str
+    :return tuple with the integer version of the seed and whether that team's seed was changed
+    """
+    seed_int = 0
+    seed_changed = False
+
+    if seed == 'full seed' or seed == 'full':
+        seed_int = 3
+    elif seed == 'half seed' or seed == 'half':
+        seed_int = 2
+    elif seed == 'free seed' or seed == 'free':
+        seed_int = 1
+
+        multiple_free_seeds = False
```

Comment:
Should be enforced by the school's handling reg rather than the excel import library. If we end up implementing this in application logic, it should go in the `models` file so it also applies to the web interface

Reply:
I've changed my implementation solely to raise a warning and encode. If you'd like the entry to get `continue`d, that's definitely feasible.